.github/workflows/composer-require-checker.yml

```
@@ -52,4 +57,4 @@ jobs:
         run: composer update --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
 
       - name: Check dependency
-        run: vendor/bin/composer-require-checker
+        run: CONFIG_PATH="$(realpath -s ${{ inputs.config }})" && test -f $CONFIG_PATH && vendor/bin/composer-require-checker --config-file=$CONFIG_PATH || vendor/bin/composer-require-checker
```

```suggestion
        run: args=(); test -z "${{ inputs.config }}" && args+=("${{ inputs.config }}"); vendor/bin/composer-require-checker ${args[@]}
```

Suggestion check on empty. Command not tested.

---

Where is `--config-file` argument name in your example?

---

Also full path is needed because plugin doesn't support relative path.

> Suggestion check on empty.

Sorry, what do you mean?

---

Discussed in PM, a simpler version exists. Updated.